code/gui/griddisplay.cpp

```diff
@@ -15,30 +15,46 @@
  * Set bounding rectangle of m_scene to m_img.size()
 */
 
+enum {
+    END_WEIGHT = -3,
+    START_WEIGHT = -2,
+    NOT_SELECTED_WEIGHT = -1,
+    DEFAULT_WEIGHT = 0,     // All selected squares should have values greater or equal to 0
+    GRID_SIZE = 10,         // Size in pixels
+    SCENE_WIDTH = 400,      // Default: 800
+    SCENE_HEIGHT = 300,     // Default: 400
+    MAX_BUTTONS_X = 50,
+    MAX_BUTTONS_Y = 50
+};
+
 const QString buttonStyle =
     // Clear button style
     "background-color: rgba(0, 0, 0, 0%);"
-    "width: 8px;"
-    "height: 8px;";
+    "width: 0px;"
```

Comment:
make these `QString`s just macros

Reply:
If I do this some of them need to be converted back to `QString`s later

Reply:
yeah that's fine, these don't need to be allocated in static memory

Reply:
you can do `QString style(STYLE_MACRO)` whenever you need it